package/gluon-config-mode-contact-info/luasrc/lib/gluon/config-mode/wizard/0500-contact-info.lua

```diff
@@ -10,12 +10,30 @@ return function(form, uci)
 		.. 'on the internet together with your node\'s coordinates.'
 	))
 
-	local o = s:option(Value, "contact", translate("Contact info"), translate("e.g. E-mail or phone number"))
-	o.default = uci:get("gluon-node-info", owner, "contact")
-	o.optional = not ((site.config_mode or {}).owner or {}).obligatory
+	local contact_field = s:option(Value, "contact", translate("Contact info"), translate("e.g. E-mail or phone number"))
+	contact_field.default = uci:get("gluon-node-info", owner, "contact")
+	contact_field.optional = not ((site.config_mode or {}).owner or {}).obligatory
 	-- without a minimal length, an empty string will be accepted even with "optional = false"
-	o.datatype = "minlength(1)"
-	function o:write(data)
+	contact_field.datatype='minlength(1)'
+	contact_field.maxlen = 140
+
+	local no_name = s:option(Flag, "no_name", translate("Anonymous node"))
+	no_name.default = false
+
+	contact_field:depends(no_name, false)
+
+	-- TODO: this should only be shown, if no_name is true
```

Comment:
Hmm, one option would be to use a value with a custom template instead of a section, but I guess it's a better idea to add dependency support for sections to gluon-web.